Gemfile.lock

```diff
@@ -25,6 +25,7 @@ GEM
     unicode-display_width (2.4.2)
 
```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍